README.md

```diff
@@ -72,6 +72,16 @@ exclude_paths: ['vendor/assets/javascripts']
 ...
 ````
 
+## Rake integration
+To use jshint in your default Rake config, just add it to the list of default tasks. For example, this configuration will run jshint in development or test environments.
+````ruby
+# your-rails-project/Rakefile
+if %w(development test).include? Rails.env
+  task(:default).clear
+  task default: [:spec, :rubocop, :jshint]
```

Comment:
Thanks for the info @threedaymonk - learned something new today :) I updated the pull request with your suggestion.